README.md

```diff
@@ -4,6 +4,10 @@
 
 This Gradle plug-in creates tasks to run [Infer](http://fbinfer.com) on Android and Java projects.
 
+## DEPRECATED
+
+This plug-in isn't used or maintained anymore. We strongly recomend migrating [NullAway](http://github.com/uber/nullaay) which is much more reliabile, significantly faster, and actively maintained.
```

Comment:
Can we just say that for null checking we recommend migrating to NullAway, without the other stuff? Infer checks other things, so NullAway doesn't subsume it. And also Infer is well maintained. I'd rather stick with something short and sweet ☺️

Reply:
Ooof, I meant that this plug-in is not currently maintained (and I think it works well for smaller projects, but I believe it has some pretty serious perf issues), thanks for catching that.